cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -304,6 +309,16 @@ func (s *CMServer) Run(_ []string) error {
 	}
 	pvRecycler.Run()
 
+	if s.EnablePersistentVolumeProvisioner {
+		if createableVolumePlugin := getCreatableVolumePluginForCloud(cloud, s); createableVolumePlugin != nil {
```

Comment:
We need to have many provisioners in a cluster.  I know of at least one party that wants it.